src/modules/Builder/components/Configuration/Configuration.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+
+import { DEBIAN_BUSTER_ARMHF, RPI_3_B, RPI_ZERO } from "./config";
+import * as RPC from "api/rpc/blackmagic";
+
+import SelectNetworkParams from "./components/Network/SelectNetworkParams";
+import SelectWirelessParams from "./components/WirelessSupport/SelectWirelessParams"
+
+export default class Configuration extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.buttonStateCallback = this.buttonStateCallback.bind(this);
+    }
+
+    executeState(builderCallback) {
+        this.builderCallback = builderCallback.bind(this);
```

Comment:
Define `builderCallback` in constructor. For example: https://github.com/tolstoyevsky/cusdeb-web-client/blob/master/src/modules/Builder/components/Initialization/Initialization.jsx#L25